tools/pyclient/README.md

```diff
@@ -13,6 +13,13 @@ pip install molgenis-emx2-pyclient
 Releases of the Molgenis EMX2 Pyclient follow the release number of the accompanying release of the Molgenis EMX2 software.
 Therefore, releases of the Pyclient are less frequent than those of EMX2 and the latest version of the Pyclient may differ from the latest version of Molgenis EMX2.
 
+#### 11.42.3

```

Comment:
This still needs to be adjusted, both version number (there is a more recent version available on pypi) as well as the changes.